arctic/store/_ndarray_store.py

```diff
@@ -129,9 +126,12 @@ def can_delete(self, version, symbol):
     def can_read(self, version, symbol):
         return version['type'] == self.TYPE
 
-    def can_write(self, version, symbol, data):
+    def can_write(self, version, symbol, data, **kwargs):
         return isinstance(data, np.ndarray) and not data.dtype.hasobject
 
+    def can_update(self, version, symbol, data, **kwargs):
```

Comment:
Yeah, I was thinking of something 'Baser'.  As the classes get bigger it becomes hard to know what the API is with `VersionStore`